quickwit/quickwit-telemetry/src/payload.rs

```diff
@@ -76,6 +76,10 @@ pub enum TelemetryEvent {
     RunService(String),
     /// EndCommand (with the return code)
     EndCommand { return_code: i32 },
+    /// Uptime timer has elapsed.
```

Comment:
We want to send an event to the telemtry server that is saying I'm `Up` or `Alive` or `Running`. 
`UptimeCheckIn` suggests some kind of registration, that's strange.

Reply:
> UptimeCheckIn suggests some kind of registration
Indeed we are doing registration. it's our interpretation that is about uptime so my thought was to have that meaning on Cloudwatch or anywhere but not here. 
Anyway, I have renamed it to `Running` to make our purpose clear.